### PR TITLE
updating readme, deleting orphaned changesets

### PR DESCRIPTION
**Pull request by ColemanRoo**
## Description
updating readme with some info about Meta models via AWS Bedrock
Deleting unused changesets

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## How Has This Been Tested?

n/a

## Checklist:

- [x] My code follows the patterns of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

## Reviewers

----

> [!IMPORTANT]
> Update `README.md` for Meta model support, increment version, and delete unused changesets.
> 
>   - **Documentation**:
>     - Update `README.md` to include support for Meta 3, 3.1, and 3.2 models via AWS Bedrock.
>   - **Versioning**:
>     - Increment version in `package.json` from 2.2.15 to 2.2.16.
>   - **Cleanup**:
>     - Delete orphaned changesets `eighty-nails-peel.md` and `shaggy-moons-dance.md`.
> 
> <sup>This description was created by </sup>[Ellipsis](https://www.ellipsis.dev?ref=RooVetGit%2FRoo-Cline&utm_source=github&utm_medium=referral)<sup> for 2ab7ea8b5e93b7b8556acbad8fa57b8f80cdece8. It will automatically update as commits are pushed.</sup>

### CODE REVIEW

**Review by mrubens — COMMENTED** [review]
For the README I've been putting Roo-specific diffs from mainline Cline in the Features at the top of the file. Can you also add to the CHANGELOG?

**Comment by ColemanRoo** [comment]
> For the README I've been putting Roo-specific diffs from mainline Cline in the Features at the top of the file. Can you also add to the CHANGELOG?

Done

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Roo Cline Changelog
 
+## [2.2.16]
+
+-   Incorporate dbasclpy's [PR](https://github.com/RooVetGit/Roo-Cline/pull/60) to add support for Meta Models via Bedrock (3, 3.1, 3.2)
```
Comment (mrubens) [inline CHANGELOG.md:5]:
```suggestion
-   Incorporate Premshay's [PR](https://github.com/RooVetGit/Roo-Cline/pull/60) to add support for Meta Models via Bedrock (3, 3.1, 3.2)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
...
+-   Unified Bedrock calls using BedrockClient and Bedrock Runtime API from [PR](https://github.com/RooVetGit/Roo-Cline/pull/60)
```
Comment (mrubens) [inline CHANGELOG.md:6]:
```suggestion
-   Incorporate Premshay's [PR](https://github.com/RooVetGit/Roo-Cline/pull/60) to add support for Meta Models via Bedrock (3, 3.1, 3.2) and unify calls using BedrockClient and Bedrock Runtime API
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,9 @@
 # Roo Cline Changelog
 
+## [2.2.16]
+
+-   Incorporate dbasclpy's [PR](https://github.com/RooVetGit/Roo-Cline/pull/60) to add support for Meta Models via Bedrock (3, 3.1, 3.2) and unified Bedrock calls using BedrockClient and Bedrock Runtime API
```
Comment (mrubens) [inline CHANGELOG.md:5]:
```suggestion
-   Incorporate Premshay's [PR](https://github.com/RooVetGit/Roo-Cline/pull/60) to add support for Meta Models via Bedrock (3, 3.1, 3.2) and unified Bedrock calls using BedrockClient and Bedrock Runtime API
```

File: README.md [hunk 4]
```diff
@@ -14,6 +14,7 @@ A fork of Cline, an autonomous coding agent, tweaked for more speed and flexibil
 - OpenRouter compression support
 - Support for newer Gemini models (gemini-exp-1206, gemini-2.0-flash-exp)
 - Runs alongside the original Cline
+- Support for Meta 3, 3.1, and 3.2 models via AWS Bedrock
```
Comment (mrubens) [inline README.md:17]:
```suggestion
- Support for newer Gemini models (gemini-exp-1206, gemini-2.0-flash-exp) and Meta 3, 3.1, and 3.2 models via AWS Bedrock
- Runs alongside the original Cline
```

**Review by mrubens — APPROVED** [review]

**Pull request merged**